projects/kit/src/lib/masks/date/utils/stringify-date.ts

```diff
@@ -0,0 +1,24 @@
+import {DEFAULT_MAX_DATE, DEFAULT_MIN_DATE} from '../../../constants';
+import {clamp, getDateSegmentValueLength, toDateString} from '../../../utils';
+import type {MaskitoDateParams} from '../date-params';
+
+export function maskitoStringifyDate(
+    date: Date,
+    {mode, separator, min = DEFAULT_MIN_DATE, max = DEFAULT_MAX_DATE}: MaskitoDateParams,
+): string {
+    const validatedDate = clamp(date, min, max);
+
+    const segmentsLength = getDateSegmentValueLength(mode);
+
+    const segments = {
+        day: validatedDate.getDate().toString().padStart(segmentsLength.day, '0'),
+        month: (validatedDate.getMonth() + 1)
+            .toString()
+            .padStart(segmentsLength.month, '0'),
+        year: validatedDate.getFullYear().toString().padStart(segmentsLength.year, '0'),
+    };
+
+    return toDateString(segments, {
+        dateMode: separator ? mode.replaceAll('/', separator) : mode,
```

Comment:
`maskitoDateOptionsGenerator` has dot as default value for `separator`.

https://github.com/taiga-family/maskito/blob/04a987a806f755d7b8cbef868269947eec4ca3ae/projects/kit/src/lib/masks/date/date-mask.ts#L14-L16

Let's do the same for `maskitoStringifyDate`:

```ts
export function maskitoStringifyDate(
    date: Date,
    {
        mode,
        separator = '.',
        min = DEFAULT_MIN_DATE,
        max = DEFAULT_MAX_DATE,
    }: MaskitoDateParams,
): string {
    // [...]

    return toDateString(segments, {
        dateMode: mode.replaceAll('/', separator),
    });
}
```